mods/ctf/ctf_modes/ctf_modebase/ranking_commands.lua

```diff
@@ -76,7 +77,20 @@ minetest.register_chatcommand("reset_rankings", {
 				return false, "The ctf_admin priv is required to reset the rankings of other players!"
 			end
 		else
+			if not allow_reset[name] then
```

Comment:
Could you also add game mode to key? Since you say ``"This will reset your stats and rankings for " .. mode_name .." mode completely."``

Reply:
:heavy_check_mark: